packages/core/package.json

```diff
@@ -67,13 +67,13 @@
   "scripts": {
     "build": "npm run build-all",
     "build-all": "run-s build:*",
-    "build:js": "grapesjs-cli build --patch=false --targets=\"> 1%, ie 11, safari 8, not dead\" --statsOutput=\"stats.json\" --localePath=\"src/i18n/locale\"",
-    "build:mjs": "BUILD_MODULE=true grapesjs-cli build --dts='skip' --patch=false --targets=\"> 1%, ie 11, safari 8, not dead\"",
+    "build:js": "node node_modules/grapesjs-cli/dist/cli.js build --patch=false --targets=\"> 1%, ie 11, safari 8, not dead\" --statsOutput=\"stats.json\" --localePath=\"src/i18n/locale\"",
```

Comment:
Note now we: 

```
node_modules/grapesjs-cli/dist/cli.js
```

Point to the workspace version that's symbolically linked in the node modules dir of the core package.